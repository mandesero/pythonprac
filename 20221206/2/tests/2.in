import asyncio

async def main(A):
    tasks, B = await mtasks(A)
    print(len(tasks))
    random.shuffle(tasks)
    await asyncio.gather(*tasks)
    return B


random.seed(228)
A = random.choices(range(10), k=40)
print(A)
B = asyncio.run(main(A))
print(*A)
print(*B)
print(*sorted(A))
print(B == sorted(A))