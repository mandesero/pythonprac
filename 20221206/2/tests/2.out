[9, 6, 2, 2, 4, 9, 6, 4, 9, 1, 8, 8, 9, 9, 1, 3, 9, 9, 9, 8, 3, 8, 4, 2, 5, 6, 9, 5, 5, 4, 8, 7, 7, 3, 2, 4, 2, 3, 2, 0]
39
9 6 2 2 4 9 6 4 9 1 8 8 9 9 1 3 9 9 9 8 3 8 4 2 5 6 9 5 5 4 8 7 7 3 2 4 2 3 2 0
0 1 1 2 2 2 2 2 2 3 3 3 3 4 4 4 4 4 5 5 5 6 6 6 7 7 8 8 8 8 8 9 9 9 9 9 9 9 9 9
0 1 1 2 2 2 2 2 2 3 3 3 3 4 4 4 4 4 5 5 5 6 6 6 7 7 8 8 8 8 8 9 9 9 9 9 9 9 9 9
True
